Makefile

```diff
@@ -68,6 +69,12 @@ modules: FORCE
 runtime: FORCE
 	cd runtime && $(MAKE)
 
+llvm-runtime-maybe: FORCE
```

Comment:
The way this is written, it is allowed to fail (the hyphen before the if stmt indicates that). For example, try building with platform=cray-xc, target_compiler=cray-prgenv-cray, and comm=gasnet (which gives substrate=aries, segment=fast, and tasks=fifo).

What about llvm-runtime-if-needed for the name?